### ISSUE TITLE
Web view does not respond after updating to v2.6.0 on Windows

### ISSUE THREAD

**Issue by jooyoungseo** [REditorSupport/vscode-R#1228]
After updating to `vscode-R` 2.6.0, either internal web view and external browser popup does not respond. for example, `gt::gt(head(iris))` does not show any web views.

I have confirmed that downgrading to v2.5.3 resolves this issue, which means that this bug has been newly introduced in the latest `vscode-R` release.

## System Info

Version: 1.72.2 (system setup)
Commit: d045a5eda657f4d7b676dedbfa7aab8207f8a075
Date: 2022-10-12T22:15:18.074Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: No

**Comment by renkun-ken** [comment]
I cannot reproduce it on Windows, macOS and Linux.

Is the session watcher able to attach to the session? What's the output of `ls.str(.vsc)`?

**Comment by jooyoungseo** [comment]
Yes, it's attached. The following is the result of `ls.str(.vsc)`:

``` r
addin_registry :  chr "C:\\Users\\jseo1005\\AppData\\Local\\Temp\\Rtmp46238h/vscode-R/addins.json"
address : function (x)  
attach : function ()  
capture_str : function (object, max.level = getOption("vsc.str.max.level", 0))  
dataview_table : function (data)  
dir_session :  chr "C:\\Users\\jseo1005\\AppData\\Local\\Temp\\Rtmp46238h/vscode-R"
dir_watcher :  chr "C:\\Users\\jseo1005\\.vscode-R"
get_column_def : function (name, field, value)  
get_response_lock : function ()  
get_response_timestamp : function ()  
get_timestamp : function ()  
globalenv_cache : <environment: 0x000002357e37b4e8> 
homedir :  chr "C:\\Users\\jseo1005"
inspect_env : function (env, cache)  
load_settings : function ()  
path_to_uri : function (path)  
pid :  int 6532
print.help_files_with_topic : function (h, ...)  
print.hsearch : function (x, ...)  
rebind : function (sym, value, ns)  
request : function (command, ...)  
request_browser : function (url, title, ..., viewer)  
request_file :  chr "C:\\Users\\jseo1005\\.vscode-R/request.log"
request_lock_file :  chr "C:\\Users\\jseo1005\\.vscode-R/request.lock"
request_response : function (command, ...)  
response_file :  chr "C:\\Users\\jseo1005\\AppData\\Local\\Temp\\Rtmp46238h/vscode-R/response.log"
response_lock_file :  chr "C:\\Users\\jseo1005\\AppData\\Local\\Temp\\Rtmp46238h/vscode-R/response.lock"
response_time_stamp :  chr ""
response_timeout :  num 5
rstudioapi_enabled : function ()  
rstudioapi_env : <environment: 0x000002357e3fa348> 
rstudioapi_util_env : <environment: 0x000002357e3fa690> 
scalar : function (x)  
settings_file :  chr "C:\\Users\\jseo1005\\.vscode-R/settings.json"
show_browser : function (url, title = url, ..., viewer = getOption("vsc.browser", "Active"))  
show_dataview : function (x, title, uuid = NULL, viewer = getOption("vsc.view", "Two"), 
    row_limit = abs(getOption("vsc.row_limit", 0)))  
show_globalenv :  logi TRUE
show_page_viewer : function (url, title = NULL, ..., viewer = getOption("vsc.page_viewer", 
    "Active"))  
show_plot :  logi TRUE
show_view :  logi TRUE
show_viewer : function (url, title = NULL, ..., viewer = getOption("vsc.viewer", "Two"))  
show_webview : function (url, title, ..., viewer)  
tempdir :  chr "C:\\Users\\jseo1005\\AppData\\Local\\Temp\\Rtmp46238h"
try_capture_str : function (object, max.level = getOption("vsc.str.max.level", 0))  
try_catch_timeout : function (expr, timeout = Inf, ...)  
update_workspace : function (...)  
use_httpgd :  logi TRUE
user_options :  chr [1:93] "add.smooth" "blogdown.author" "blogdown.ext" ...
wd :  chr "C:/work/imls_bvi_a11y_making"
workspace_file :  chr "C:\\Users\\jseo1005\\AppData\\Local\\Temp\\Rtmp46238h/vscode-R/workspace.json"
workspace_lock_file :  chr "C:\\Users\\jseo1005\\AppData\\Local\\Temp\\Rtmp46238h/vscode-R/workspace.lock"
```

**Comment by renkun-ken** [comment]
If you "Toggle Developer Tools", do you see a lot of warnings or errors regarding vscode-R extension?

**Comment by jooyoungseo** [comment]
Here is the result:

![image](https://user-images.githubusercontent.com/19754711/197343628-cbe58fde-baf4-4559-8af1-421fe8d1e954.png)

**Comment by renkun-ken** [comment]
What's your settings regarding `r.*`? From the console log in the developer tools, the webview request has `"viewer": false` which means WebView viewer inside vscode is disabled.

**Comment by jooyoungseo** [comment]
@renkun-ken -- I have found the following option in `.Rprofile` is the cause of the issue.

``` r
options(vsc.viewer = FALSE)
```

FYI, this option works for `vscode-R` v2.5.3. I would like to keep using that option because I prefer to open html content in an external browser.

**Comment by renkun-ken** [comment]
Now I could reproduce it with the following:

```r
options(vsc.viewer = FALSE)
gt::gt(iris)
```

The html file should be open externally in a web browser. Let me take a closer look at this.

**Comment by renkun-ken** [comment]
Should be fixed via 137f6c0.

**Comment by jooyoungseo** [comment]
Thank you very much for the quick fix, @renkun-ken !
